src/com/owncloud/android/lib/common/OwnCloudClient.java

```diff
@@ -199,62 +200,85 @@ public int executeMethod(HttpMethodBase method, int readTimeout, int connectionT
 
     /**
      * Requests the received method.
-     *
+     * <p>
      * Executes the method through the inherited HttpClient.executedMethod(method).
      *
-     * @param method                HTTP method request.
+     * @param method HTTP method request.
      */
     @Override
     public int executeMethod(HttpMethod method) throws IOException {
-        try {
-            // Update User Agent
-            HttpParams params = method.getParams();
-            String userAgent = OwnCloudClientManagerFactory.getUserAgent();
-            params.setParameter(HttpMethodParams.USER_AGENT, userAgent);
+        // Update User Agent
+        HttpParams params = method.getParams();
+        String userAgent = OwnCloudClientManagerFactory.getUserAgent();
+        params.setParameter(HttpMethodParams.USER_AGENT, userAgent);
 
-            Log_OC.d(TAG + " #" + mInstanceNumber, "REQUEST " +
-                    method.getName() + " " + method.getPath());
+        preventCrashDueToInvalidPort(method);
 
-//	        logCookiesAtRequest(method.getRequestHeaders(), "before");
-//	        logCookiesAtState("before");
-            method.setFollowRedirects(false);
+        Log_OC.d(TAG + " #" + mInstanceNumber, "REQUEST " +
+            method.getName() + " " + method.getPath());
 
-            int status = super.executeMethod(method);
+        //logCookiesAtRequest(method.getRequestHeaders(), "before");
```

Comment:
To uncomment it in development context when things go really, really bad. Trust me, let's keep those. See my last comment, too.